src/server/handlers/manager/mod.rs

```diff
@@ -25,9 +26,16 @@ pub mod topdown_executed;
 pub mod whitelist;
 
 pub(crate) fn check_subnet(subnet: &Subnet) -> Result<()> {
-    if subnet.auth_token().is_none() {
-        log::error!("subnet {:?} does not have auth token", subnet.id);
-        return Err(anyhow!("Internal server error"));
+    match &subnet.config {
+        SubnetConfig::Fvm(config) => {
+            if config.auth_token.is_none() {
+                log::error!("subnet {:?} does not have auth token", subnet.id);
+                return Err(anyhow!("Internal server error"));
+            }
+        }
+        SubnetConfig::Evm(_) => {
+            // TODO: add more checks later
```

Comment:
What kind of checks are you thinking of?

Reply:
Ensure there are accounts configured (ofc after we have added the wallet store)?